config/stage.json

```diff
@@ -34,7 +34,7 @@
 		"external-media": true,
 		"external-media/google-photos": true,
 		"external-media/free-photo-library": true,
-		"google-my-business": false,
```

Comment:
stage should be very very close to production, I don't think it's a good idea to activate `google-my-business` there